Yashodha's org/Fare calculator API/service.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina/http;
+import ballerinax/java.jdbc;
+import ballerina/io;
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+
+type FlightDetails record {
+    string flightNumber;
+    string airline;
+    string flightDate;
+    string origin;
+    string destination;
+    int available;
+    int totalCapacity;
+    int baseFare;
+};
+
+service /fareCal on new http:Listener(9090) {
```

Comment:
Can we reuse the same listener as you want this service also on port 9090?

Reply:
Also the service path is not descriptive and feels like it's at a half way. To invoke following resource, the request path is `/fareCal/fareCalculator`. We can structure it properly

Either use `/fareCalculator` as service path and resource path as `.` 
or, use `/` as service path and resource path as `fareCalculator`